contrib/backporting/cherry-pick

```diff
@@ -18,7 +18,7 @@ cherry_pick () {
   echo "[ upstream commit $FULL_ID ]" >> $TMPF
   git format-patch -1 $FULL_ID --stdout | sed -n '/^$/,$p' >> $TMPF
   echo "Applying: $(git log -1 --oneline $FULL_ID)"
-  git am --quiet -3 --signoff $TMPF
+  git am --quiet -3 --signoff $TMPF || rc=$?; rm $TMPF; exit $rc
```

Comment:
Not sure what a cleaner way to achieve this would look like. @qmonnet Any ideas?

Reply:
Use a `trap` on `EXIT` to call a clean-up function that would remove the file if it exists? (Grep for `trap` in shell scripts, e.g. `contrib/scripts/cilium-up.sh`.)

Reply:
Thanks!